docs/gallery/domain/plot_icephys.py

```diff
@@ -97,6 +97,7 @@
 
 # Import additional core datatypes used in the example
 from pynwb.core import DynamicTable, VectorData
+from pynwb.base import TimeSeriesReference, TimeSeriesReferenceVectorData
```

Comment:
Just as side note, technically we could use `TimeSeriesReferenceVectorData.TIME_SERIES_REFERENCE_TUPLE` instead, but I think importing `TimeSeriesReference` directly looks cleaner.